.github/workflows/build.yml

```diff
@@ -8,15 +8,19 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          #- os: windows-2019
-          #  vcpkg_path: C:\mixxx-vcpkg
-          #  vcpkg_bootstrap: .\bootstrap-vcpkg.bat
-          #  vcpkg_triplet: x64-windows
-          # check_disk_space: Get-PSDrive
+          - os: windows-2019
```

Comment:
You have removed libmad and libid3tag for Windows.

Reply:
This commit is outdated.
Please compare to the final result.

Reply:
This PR is difficult to review because of the intermediate changes that got reverted later. Can you squash it down to a few commits, perhaps:
1. portmidi changes
2. macOS 10.12 target changes
3. GH Actions workflow changes